libs/styles/src/lib/scss/components/_datepicker.scss

```diff
@@ -0,0 +1,141 @@
+$datepicker-border-color: #9799A4;
+$datepicker-border-radius: 4px;
+$datepicker-font-size: 16px;
+$datepicker-width: 125px;
+$datepicker-height: 40px;
+$datepicker-color: #34394C;
+
+@mixin cvi-datepicker($base: &) {
+  --comment: 'A datepicker';
+
+  @at-root {
+    #{$base}__input {
+      &-container {
+        border: 1px solid $datepicker-border-color;
+        border-radius: $datepicker-border-radius;
+        display: flex;
+        align-items: center;
+        width: $datepicker-width;
+        height: $datepicker-height;
+
+        &-focused {
+          border-color: get-color(sapphire-blue-10);
+        }
+      }
+
+      &-field {
+        font-size: $datepicker-font-size;
+        border-left: 1px;
```

Comment:
This declaration is invalid and ignored in browser (`border` [requires at least two values](https://developer.mozilla.org/en-US/docs/Web/CSS/border)).